src/pages/ProductList/components/GoodList/GoodList.js

```diff
@@ -3,45 +3,96 @@ import { useNavigate } from 'react-router-dom';
     <div className="goodList">
-      {list.map(({ id, rendingImage, hoverImage, title, price }) => {
-        return (
-          <div key={id} className="goodListItem">
-            <div className="image">
-              <img
-                className="rendingImage"
-                src={rendingImage}
-                alt="rendingImage"
-              />
-              <img className="hoverImage" src={hoverImage} alt="hoverImage" />
-              <AiOutlineShoppingCart
-                className="hoverCart"
-                onClick={() => {
-                  navigate('/cart');
-                }}
-              />
-            </div>
-            <p className="item">{title}</p>
-            <p className="price"> {price} WON</p>
-            <div>
-              <Heart />
-              <span className="count">1</span>
-            </div>
-          </div>
-        );
-      })}
+      {goodList &&
+        goodList.map(
+          ({
+            productId,
+            productImage,
+            productName,
+            productPrice,
+            discount_rate,
+          }) => {
+            return (
+              <div key={productId} className="goodListItem">
+                <div className="image">
+                  <img
+                    className="rendingImage"
+                    src={productImage[0]}
+                    alt="rendingImage"
+                  />
+                  <img
+                    className="hoverImage"
+                    src={productImage[1]}
+                    alt="hoverImage"
+                  />
+                  <AiOutlineShoppingCart
+                    className="hoverCart"
+                    onClick={() => {
+                      navigate('/cart');
+                    }}
+                  />
+                </div>
+                <p className="item">
+                  {productName}
+                  <span className="heartCount">
+                    <Heart />
+                    <span className="count">1</span>
+                  </span>
+                </p>
+
+                <div>
+                  {/* <p className="price"> {productPrice} WON</p> */}
+                  {discount_rate === 0 ? (
+                    <p className="normalPrice">
+                      {' '}
+                      {productPrice.toLocaleString()} WON
+                    </p>
+                  ) : (
+                    <p>
+                      <p className="price">
+                        {' '}
+                        {productPrice.toLocaleString()} WON
+                      </p>
+                      <p className="discount">
+                        <span className="discountPrice">
+                          {(productPrice * discount_rate).toLocaleString()} WON{' '}
+                        </span>
+
+                        <span className="discountRate">
+                          {`(${Math.floor((1 - discount_rate) * 100)}%`}{' '}
+                          <BsArrowDown />
```

Comment:
```suggestion
                      {productPrice.toLocaleString()} WON
                    </p>
                  ) : (
                    <p>
                      <p className="price">
                        {productPrice.toLocaleString()} WON
                      </p>
                      <p className="discount">
                        <span className="discountPrice">
                          {(productPrice * discount_rate).toLocaleString()} WON
                        </span>

                        <span className="discountRate">
                          {`(${Math.floor((1 - discount_rate) * 100)}%`}
                          <BsArrowDown />
```
{" "}
빈 문자열이여서 삭제 해주셔도 됩니다

Reply:
완료했습니다.